runtime/lua/vim/lsp/handlers.lua

```diff
@@ -76,12 +90,27 @@ M['textDocument/publishDiagnostics'] = function(...)
   return require('vim.lsp.diagnostic').on_publish_diagnostics(...)
 end
 
+--- LSP method textDocument/references
+--@param _ (not used)
```

Comment:
do not document `_` params, there is no need.

Reply:
fixed